Dockerfile

```diff
@@ -7,9 +7,11 @@
 FROM python:3.6-slim
 
 ENV TERM=xterm
+
 RUN apt-get update && \
     apt-get install -y vim-tiny && \
-    pip install --upgrade pip
+    pip install --upgrade pip && \
+    pip install htcondor==8.9.6 retrying
```

Comment:
Do we need to have these libraries installed at this specific point in the Dockerfile, maybe because of the deb packages? Otherwise, we could just continue setting them in [`setup.py`](https://github.com/reanahub/reana-job-controller/blob/35c4fc8ab3c79a78e8709475e80ef2fd6b80bb85/setup.py#L55) so they will be installed at [this point](https://github.com/reanahub/reana-job-controller/blob/35c4fc8ab3c79a78e8709475e80ef2fd6b80bb85/Dockerfile#L60), and later available in the code .

Reply:
You're right, these should be handled in `setup.py`, I'll clean these up.